pkgs/development/libraries/jemalloc/default.nix

```diff
@@ -2,18 +2,25 @@
 
 stdenv.mkDerivation rec {
   name = "jemalloc-${version}";
-  version = "4.3.1";
+  version = "4.5.0";
 
   src = fetchurl {
     url = "https://github.com/jemalloc/jemalloc/releases/download/${version}/${name}.tar.bz2";
-    sha256 = "12r71i8nm3vwz21fc16rwbb0pwcg5s05n1qg3rwl2s85v0x1ifzp";
+    sha256 = "9409d85664b4f135b77518b0b118c549009dc10f6cba14557d170476611f6780";
   };
 
   # By default, jemalloc puts a je_ prefix onto all its symbols on OSX, which
   # then stops downstream builds (mariadb in particular) from detecting it. This
   # option should remove the prefix and give us a working jemalloc.
   configureFlags = stdenv.lib.optional stdenv.isDarwin "--with-jemalloc-prefix=";
 
+  preInstall = ''
+    substituteInPlace bin/jeprof --replace '#! /usr/bin/env perl' '#!/run/current-system/sw/bin/perl'
```

Comment:
Indeed this should point to Perl in the store and not to the symlink in `current-system`.
But do we want jemalloc to depend on Perl ?  :/